react/week4/git-api/src/components/FormComponent.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react"
+import { UserContext } from "./UserContext"
+
+const FormComponent = ({ input, setInput, loading }) => {
+  const users = useContext(UserContext)
+  console.log(users)
+  const handleChange = e => {
+    console.log(e.target.value)
+    setInput(e.target.value)
+  }
+  return (
+    <div>
+      <form>
+        <p>Search for git users:</p>
+        <input type="text" onChange={handleChange} placeholder="search" />
+      </form>
+      {users.length >= 1 ? (
+        <ul>
+          {!loading ? (
+            users.map(user => <li>{user.login}</li>)
```

Comment:
Don't forget to give your list items a key (which can be the index). This way React can keep better track of changes in the list. 
https://reactjs.org/docs/lists-and-keys.html